services/systemd/swaync.service.in

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Swaync notification daemon
+Documentation=https://github.com/ErikReider/SwayNotificationCenter
+PartOf=graphical-session.target
+After=graphical-session.target
+Requisite=graphical-session.target
```

Comment:
Removing this line seems to fix the systemd error

Reply:
I put it there to only start the service if graphical-session was registered as started.
In teory, the condition checking the existance of WAYLAND_DISPLAY should do just fine, but I think Requisite is better suited and more explicit as a directive.
Still, at least WAYLAND_DISPLAY should remain, so it is not started on X11 by accident (it's only intended for wayland sessions, right?).

Regarding Requisite, the writers of the [systemd unit shipped by waybar](https://github.com/Alexays/Waybar/blob/master/resources/waybar.service.in) seem to think the same way.

I probably didn't notice any issue because I followed the [community guide on Sway's wiki to better integrate with systemd](https://github.com/swaywm/sway/wiki/Systemd-integration). Due to that, I launch
```
exec systemctl --user start sway-session.target
```
as the last operation in sway's config files.
That target binds to graphical-session and so it registers as started.

Reply:
I know it's a little controversial, but if one is interested in combining their wayland session with systemd services, they really should do it right, at least for basic targets. Am I a little too harsh?

Reply:
I'm not really that familiar with the situation so I can't really say anything about it. We should keep the `Requisite` set to `graphical-session.target`